src/visualizer/visualizer.js

```diff
@@ -16,11 +16,25 @@ const week = "week";
 const month = "month";
 const year = "year";
 
+const getMaxPriorityElement = prioritized => {
+    const max_priority = [prioritized[0]]
```

Comment:
Please use camel case instead of snake case

Reply:
please dont forget to put `;` as well

Reply:
After reading down into the whole function, i would suggest to change the variable name into `topPriorities` since we put all of max level priority deadlines there

Reply:
Done.